packages/cactus-plugin-keychain-google-sm/src/main/typescript/plugin-keychain-google-sm-remote-adapter.ts

```diff
@@ -0,0 +1,155 @@
+import type { Server } from "http";
```

Comment:
@jagpreetsinghsasan Sorry, /facepalm moment here. We do not need the remote adapter for this package because the implementation runs natively in NodeJS anyway, so it's pointless. The reason why we had the remote adapter for the vault keychain because that one originally was written in Rust and so there it is necessary, but otherwise no need, so this file can be deleted altogether, apologies for not realizing this earlier.